crypto/secp256k1/secp256k1.go

```diff
@@ -4,25 +4,29 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"crypto/subtle"
+	"errors"
```

Comment:
`crypto/secp256k1/secp256k1.go` is used as `nocgo`, right?
In BSC side, `nocgo` is not used, BSC use  `secp256k1_nocgo.go`, which is removed in this PR, is it reasonable?